spec/mocks/server.js

```diff
@@ -0,0 +1,68 @@
+const { ParseServer } = require('parse-server');
+const MockEmailAdapterWithOptions = require('./MockEmailAdapterWithOptions');
+const { makeS3Adapter } = require('./s3adapter-v2');
+
+const port = 1327;
+const mountPath = '/api/parse';
+const serverURL = 'http://127.0.0.1:1327/api/parse';
+
+const PARSE_APP_ID = 'app-id';
+const PARSE_MASTER_KEY = 'master-key';
+
+const S3Adapter = makeS3Adapter();
+
+const defaultConfiguration = {
+  databaseURI: 'mongodb://127.0.0.1:27017/s3-adapter',
+  appId: PARSE_APP_ID,
+  masterKey: PARSE_MASTER_KEY,
+  serverURL,
+  liveQuery: {
+    classNames: [],
+  },
+  startLiveQueryServer: true,
+  verbose: false,
+  silent: true,
+  fileUpload: {
+    enableForPublic: true,
+    enableForAnonymousUser: true,
+    enableForAuthenticatedUser: true,
+  },
+  revokeSessionOnPasswordReset: false,
+  allowCustomObjectId: false,
+  allowClientClassCreation: true,
+  encodeParseObjectInCloudFunction: true,
```

Comment:
Are all these options needed? Maybe you could remove all the ones that are not needed.

Reply:
Sure